common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -801,6 +875,13 @@ public void run() {
         });
     }
 
+    /**
+     * An interface that is called with a long argument.
+     */
+    protected interface LongConsumer {
```

Comment:
make a top level interface in `io.netty.util` to increase visibility (easier to find/replace later)